src/main/resources/fabric.mod.json

```diff
@@ -29,7 +29,8 @@
     "minecraft": ">=${minecraft_version}",
     "fabricloader": "*",
     "fabric-api": "*",
-    "cloth-config": "*"
+    "cloth-config": "*",
+    "sodium": ">=0.7.0"
```

Comment:
That would have required Sodium to always be installed.
I updated the file to specify the mod breaking with Sodium <0.7.0 instead